components/mpas-albany-landice/driver/glc_comp_mct.F

```diff
@@ -1592,8 +1608,8 @@ subroutine glc_export_mct(g2x_g, errorCode)
             g2x_g % rAttr(index_g2x_Sg_icemask_coupled_fluxes, n) = g2x_g % rAttr(index_g2x_Sg_icemask, n)
 
             ! Add masks used for ice shelf basal melt calculation
-            g2x_g % rAttr(index_g2x_Sg_icemask_floating, n) = li_mask_is_floating_ice_int(cellMask(i))
-            g2x_g % rAttr(index_g2x_Sg_icemask_grounded, n) = li_mask_is_grounded_ice_int(cellMask(i))
+            g2x_g % rAttr(index_g2x_Sg_icemask_below_sea_level, n) = belowSeaLevelMask * li_mask_is_ice_int(cellMask(i))
+            g2x_g % rAttr(index_g2x_Sg_icemask_floating, n) = belowSeaLevelMask * li_mask_is_floating_ice_int(cellMask(i))
```

Comment:
We may want to change this to floating locations below sea level and connected to the open ocean (i.e. ignoring subglacial lakes).  We've had to implement such a mask elsewhere in MALI to deal with unintentionally applying ocean-driven melt rates at lakes.